components/tracking/install.sh

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Codacy found an issue: [Tips depend on target shell and yours is unknown. Add a shebang.](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5609236)